src/main/resources/model/authority.json

```diff
@@ -323,8 +323,9 @@
       "default": "NULL"
     },
     "naturalId": {
-      "index": "keyword",
-      "showInResponse": [ "search", "browse" ]
+      "index": "keyword_lowercase",
+      "showInResponse": [ "search", "browse" ],
+      "searchTermProcessor": "naturalIdSearchTermProcessor"
     },
```

Comment:
Why there a need for processor that makes value lowecase if we already use keyword-lowercase?